ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx

```diff
@@ -5,24 +5,25 @@ import {
   makeStyles,
   Theme,
   useTheme
-} from '@material-ui/core';
-import MoreVertIcon from '@material-ui/icons/MoreVert';
-import { inject, observer } from 'mobx-react';
-import React, { ChangeEvent, useState } from 'react';
-import { useTranslation } from 'react-i18next';
+} from "@material-ui/core";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+import { inject, observer } from "mobx-react";
+import React, { ChangeEvent, useState } from "react";
+import { useTranslation } from "react-i18next";
 import {
   statusColors,
   primaryLighter
-} from '../../assets/themes/exilence-theme';
-import { ITableItem } from '../../interfaces/table-item.interface';
-import { AccountStore } from '../../store/accountStore';
-import { SignalrStore } from '../../store/signalrStore';
-import { UiStateStore } from '../../store/uiStateStore';
-import { mapPricedItemToTableItem } from '../../utils/item.utils';
-import ItemTableFilter from './item-table-filter/ItemTableFilter';
-import ItemTable, { Order } from './ItemTable';
-import FilterListIcon from '@material-ui/icons/FilterList';
-import ItemTableMenuContainer from './item-table-menu/ItemTableMenuContainer';
+} from "../../assets/themes/exilence-theme";
```

Comment:
Would love if you could change your formatting settings to match ours. We use pretteir with single quotes

Reply:
Fixed the quotes to match.